grpc/server/src/main/java/io/helidon/grpc/server/MethodDescriptor.java

```diff
@@ -246,6 +276,15 @@ public MetricType metricType() {
             return metricType(MetricType.INVALID);
         }
 
+        @Override
+        public Builder<ReqT, ResT> marshallerSupplier(MarshallerSupplier supplier) {
+            if (supplier != null) {
+                descriptor.setRequestMarshaller(supplier.get(requestType))
+                          .setResponseMarshaller(supplier.get(responseType));
```

Comment:
Hmm... This assumes certain ordering between request/responseType being set before `marshallerSupplier` is called. I'm not sure I like that.

IIRC, you are passing request/responseType as arguments to the `unary` and other method-definition methods on `ServiceDescriptor.Builder`, which allows this to work but I'm not sure it's the right approach. Shouldn't they be set optionally, within `Config` consumer as well?

May be best that we do an interactive Zoom session to sort this out and make a decision.